lib/diagnostic.rb

```diff
@@ -35,7 +45,7 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
+dave = Person.new(name: "Dave", age: 32, location: "Ohio")
 Response.dave = dave
 ##
 
```

Comment:
-.5 Remember to read the whole question!
How would we change the location property to "Somerville"?